### PR TITLE
RD-3416: set sdk version header based on value in composer.json

### PR DESCRIPTION
**Pull request by gianluca-token**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "tokenio/sdk",
   "description": "PHP SDK for interacting with the Token System",
-  "version": "1.0.0-beta-4",
+  "version": "1.1.0",
```
Comment (yuhanniu) [inline composer.json]: 1.0.0?

File: composer.json [hunk 2]
```diff
@@ -25,7 +25,8 @@
     "stephenhill/base58": "~1.0",
     "grpc/grpc": "^1.20.0",
     "google/protobuf": "^3.6.1",
-    "jms/serializer": "1.7.1"
+    "jms/serializer": "1.7.1",
+    "composer/composer": "dev-master"
```
Comment (yuhanniu) [inline composer.json]: do they not have a published version?

File: composer.json (hunk 1 above)
Comment (gianluca-token) [reply composer.json]: it already exists https://github.com/tokenio/sdk-php/commit/446d8b2b68ab3a7adf675e15f96ac407c668fded

Comment (yuhanniu) [reply composer.json]: never published

**Pull request merged**